### PR TITLE
Fixed errors and cleaned descriptions

### PR DESCRIPTION
**Pull request by kyscg**

### CODE REVIEW

**Comment by kyscg** [comment]
Fixed issue in #125

File: tensorflow_datasets/image/celebahq.py [hunk 1]
```diff
@@ -43,7 +43,8 @@
 High-quality version of the CELEBA
 dataset, consisting of 30000 images in 1024 x 1024 resolution.
 
-WARNING: this dataset currently requires you to prepare images on your own.
+WARNING: This dataset currently requires you to prepare images on your own. Tensorflow also provides an image generator
```
Comment (cyfra) [inline tensorflow_datasets/image/celebahq.py]: The link you added is to Tensor2Tensor library.

Not sure what you meant by 'provides an image generation'?
That library unfortunately also requires you to prepare the data yourself.

Comment (kyscg) [reply tensorflow_datasets/image/celebahq.py]: I meant to provide a method for those interested in image generation, does it seem unnecessary?

Comment (cyfra) [reply tensorflow_datasets/image/celebahq.py]: I think that it is not necessary.

**Pull request merged**